test/TickMath.starknet.ts

```diff
@@ -163,16 +163,16 @@ describe('TickMath', () => {
         it('is at most off by 1', async () => {
           const jsResult = new Decimal(ratio.toString()).div(new Decimal(2).pow(96)).pow(2).log(1.0001).floor()
           const result = await tickMath.getTickAtSqrtRatio_4f76c058(new BN(ratio.toString()).toTwos(24).toString())
```

Comment:
Since the values are not negative, we can use `getTickAtSqrtRatio_4f76c058(ratio.toString())`